template/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.8)
-set(32BLIT_PATH "../" CACHE PATH "Path to 32blit.cmake")
 project (game)
+set(32BLIT_PATH "../" CACHE PATH "Path to 32blit.cmake")

```

Comment:
In case anyone is wondering, this change is because the toolchain isn't parsed until the `project()` call... Also, this works because `set(... CACHE ...)` doesn't override existing variables.